frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -176,20 +177,19 @@ export class MineTailingsInfo extends Component {
               <Col span={6} />
             </Row>
           ))}
-          <div className="center">
-            <Button
+          <div className="center">            
```

Comment:
Do you have Prettier enabled in your editor?

Reply:
yeah, enabled the on-save earlier today.

Some filed may not have been saved after adding it.